Psyche/Assets/Scripts/GameStates/GameStateManager.cs

```diff
@@ -240,15 +312,15 @@ public IEnumerator Warp()
         yield return new WaitForSeconds(1.2f);
 
         //check if the player is at the starting point
-        if (startPoint.Equals(respawnPoint))
+        if (StartPoint.Equals(RespawnPoint))
         {
             //UnityEngine.SceneManagement.SceneManager.LoadScene(UnityEngine.SceneManagement.SceneManager.GetActiveScene().name);

```

Comment:
I think this commented out code can be removed. Other than that I don't think I see any other errors.

Reply:
resolved